starter-code/src/StackDataStructure.js

```diff
@@ -1,2 +1,72 @@
-function StackDataStructure () {
-}
+class StackDataStructure {
+  constructor() {
+    this.stackControl = [];
+    this.MAX_SIZE = 10;
+  }
+  isEmpty() {
+    if (this.stackControl.length === 0) {
+      return true;
+    } else {
```

Comment:
no es necesario el ``èlse``` puedes hacer:
```js
    if (this.stackControl.length === 0) {
       return true;
    } 
    return false;

```